modules/prebidServerBidAdapter/index.js

```diff
@@ -543,6 +545,17 @@ export const processPBSRequest = hook('sync', function (s2sBidRequest, bidReques
   }
 }, 'processPBSRequest');
 
+function emitNoBid(bid) {
+  const status = bid.nonbid.statuscode;
+  if (status > 0 && status < 100) {
+    events.emit(CONSTANTS.EVENTS.NO_BID, bid);
+  } else if (status > 99 && status < 400) {
+    events.emit(CONSTANTS.EVENTS.BIDDER_ERROR, bid);
+  } else if (status === undefined) {
+    logError('No bid not emitted. No status defined');
+  }
```

Comment:
Do we not want to use the [BID_REJECTED](https://github.com/prebid/Prebid.js/blob/master/src/constants.json#L32) event for any of the "REQUEST_BLOCKED" (200-299) or "RESPONSE_REJECTED" (300-400) statuses from the [Status Code Guidance](https://github.com/prebid/prebid-server/issues/2367#issuecomment-1299097069)?   Or is the "BIDDER_ERROR" event enough?  Or maybe add another "BID_BLOCKED" event to distinguish between **blocked** and **rejected** bids?

Reply:
Also, I think we need to update the Magnite Analytics Adapter to handle these events (may require more discussion)

Reply:
I think we're planning to handle the analytics in a separate PR.  I'm not sure who to ask about which event to emit. We don't need to worry too much about stepping on other adapters' toes since anyone hearing this event has to opt in in their s2s conf.

Reply:
@spotxslagle can you mark this second pr as solving #9248?